### ISSUE TITLE
[geometry-1] DOMMatrix with is2D should not store -0 for "3d" members

### ISSUE THREAD

**Issue by zcorpan** [w3c/fxtf-drafts#394]
See https://bugzilla.mozilla.org/show_bug.cgi?id=1562214#c6

> Another option is to change the spec to match what Gecko does, or to set is2D to false when assigning -0.

cc @bzbarsky @dirkschulze @chrishtr @smfr

**Comment by tabatkins** [comment]
Yeah, I think "set is2D to false when assigning -0 to one of the 3d-only matrix members" is reasonable. There's no reasonable 2d-related case that should result in a -0 being set there.

**Comment by zcorpan** [comment]
@smfr does this sound good to you?

Here's a demo that might make it clearer what this issue is about:
http://software.hixie.ch/utilities/js/live-dom-viewer/saved/7843
